### DISCUSSION TITLE
Text Highlight

### DISCUSSION THREAD

**Discussion by FullStackScott** [atom/atom#24936] [General]
Good Day Alll.

Sometimes my highlighted text remains highlighted in the editor even after a click further down/up my php page(s). I replaced my highlight-selected package, but it still happens.  This started sometime last week.

Any help appreciated

FullStackScott

**Comment by jaygriz** [comment]
hit esc key?

**Reply by FullStackScott** [reply]
nope, still highlight.  This happens after Ctrl-C

**Reply by FullStackScott** [reply]
After I ctrl-c highlighted text remains, press enter 2x, highlight remains, ctrl-v and pasted text is highlighted, click away from highlighted, still highlighted.

**Comment by FullStackScott** [comment]
I reinstalled 1.60.0 atom.io, no joy
